Add new OSTree properties and add brief usage path (#3016)
theforeman/foreman-documentation: guides/common/modules/proc_importing-ostree-content.adoc

Clear this field if the repository does not require authentication.
. In the *Upstream Password* field, enter the corresponding password for the upstream repository.
. In the *Upstream Authentication Token* field, enter the corresponding token for the upstream repository.

> . Optional: In the *Exclude Refs* field, enter a list of OSTree head refs seperated by commas to exclude from importing to {Project}.

Reviewer: ```suggestion
. Optional: In the *Exclude Refs* field, enter a list of OSTree head refs separated with commas to exclude from importing to {Project}.
```

> The exclude will be evaluated after the includes.

Reviewer: ```suggestion
The excludes are evaluated after the includes.
```

> . Optional: In the *Include Refs* field, enter a list of OSTree head refs seperated by commas to include from importing to {Project}.

Reviewer: ```suggestion
. Optional: In the *Include Refs* field, enter a list of OSTree head refs separated with commas to include in importing to {Project}.
```
